sdk/servicebus/Microsoft.Azure.ServiceBus/tests/ServiceBusConnectionStringBuilderTests.cs

```diff
@@ -191,5 +191,15 @@ public async Task NonAmqpUriSchemesShouldWorkAsExpected()
                 await receiver.CloseAsync();
             });
         }
+
+        [Fact]
+        public void InvalidAadConnectionStringTest()
```

Comment:
I'd suggest we consider avoiding the abbreviation here and go with something like `InvalidAzureActiveDirectoryConnectionStringFailsValidation` for clarity.   The more explicit the name, the more helpful it is to troubleshoot from the build dashboard for those without deep familiarity with the code.